lib/datasource/conan/index.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { getPkgReleases } from '..';
+import * as httpMock from '../../../test/http-mock';
+import { loadJsonFixture } from '../../../test/util';
+import * as loose from '../../versioning/loose';
```

Comment:
Why do you use loose versioning instead of default conan versioning?

Reply:
that is a really good question... That's probably a hold out